indico/modules/auth/util.py

```diff
@@ -69,3 +70,12 @@ def url_for_login(next_url=None):
 
 def url_for_logout(next_url=None):
     return url_for('auth.logout', next=next_url)
+
+
+def url_for_register(next_url=None):
+    local_identities_enabled = Config.getInstance().getLocalIdentities()
```

Comment:
used only once, so you could use `if Config.getInstance().getLocalIdentities():` directly